templates/catalog/_partials/miniatures/product.tpl

```diff
@@ -53,9 +53,15 @@
 
         <div class="highlighted-informations{if !$product.main_variants} no-variants{/if}">
           {block name='quick_view'}
-            <a class="quick-view js-quick-view" href="#" data-link-action="quickview">
-              <i class="material-icons search">&#xE8B6;</i> {l s='Quick view' d='Shop.Theme.Actions'}
-            </a>
+            {if $product.customization_required}
+              <a class="see-product" href="{$product.url}">
+                <i class="material-icons search">&#xE8B6;</i> {l s='See product' d='Shop.Theme.Actions'}
```

Comment:
```suggestion
                <i class="material-icons search">&#xE8B6;</i> {l s='View product' d='Shop.Theme.Actions'}
```

I think it should be `View product`, but maybe @l-delin could help us out

Reply:
I don't think this is a wording issue, but a UX one. Wouldn't it be better to get rid of the "Quick view" hovering menu altogether since it's useless in this case?
If there really is a need for a new wording, I think "See options" is broad enough to work with any product.

```suggestion
                <i class="material-icons search">&#xE8B6;</i> {l s='See options' d='Shop.Theme.Actions'}
```

Reply:
I agree with Léa. I don’t think we need to replace the « quick view » wording. We should delete it completely. The user will just have to click on the image to display the product page, as usual. Otherwise clicking on the image and on the wording will give the same result.

Reply:
This is my bad, I should have explained why I did replace the wording, it was to avoid a breaking change 

If we remove the wording, we need to remove the whole quickview hover UI in case both combinations and customizations aren't set

If I do this, peoples who added something else than combinations/quickview inside this panel will see it disappear but they want to keep it like it is

Reply:
In that case, the suggestion I made previously is the best option to replace  "see product"